packages/mobile/test/values.ts

```diff
@@ -51,6 +51,15 @@ export const mockRecipientWithPhoneNumber: RecipientWithMobileNumber = {
   e164PhoneNumber: mockE164Number,
 }
 
+export const mockRecipientWithContact: RecipientWithContact = {
```

Comment:
Why add this instead of using the existing `mockRecipient` above?

Reply:
good catch, didn't see that one!